### DISCUSSION TITLE
Score import glitch

### DISCUSSION THREAD

**Discussion by JakeBlade2017** [ppy/osu#14354] [Q&A] [answered]
**When i import my scores from osu! estable the profile photo doesn't load and the country flag too**

![osu_2021-08-16_16-32-50](https://user-images.githubusercontent.com/68071816/129646493-b5dc83d4-dda3-4457-a20d-ecc3f3efb824.jpg)

**Comment by frenzibyte** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/issues/5966.
